steps/assets/openstack/main.tf

```diff
@@ -33,15 +59,15 @@ module assets_base {
   tectonic_versions                = "${var.tectonic_versions}"
 }
 
+data "ignition_user" "ssh_authorized_key" {
+    name = "core"
+    ssh_authorized_keys = ["${data.openstack_compute_keypair_v2.openstack_ssh_key.public_key}"]
```

Comment:
I wonder if we can simplify this a bit by adding support for metadata to the ignition openstack provider?

E.g atm it only reads the user_data from the metadata service/config-drive not http://169.254.169.254/openstack/2012-08-10/meta_data.json which will contain some predictable data like the hostname and ssh key?

If we added that it'd also make the initial boot of nodes with empty user_data easier as at least the ssh keys etc could be set up without explicit configuration.